src/lib/bootloader/language.rb

```diff
@@ -0,0 +1,19 @@
+require "cfa/base_model"
+require "cfa/augeas_parser"
+
+module Bootloader
+  # read only model to get system language from sysconfig
+  # Uses CFA base model with augeas parser
+  class Language < CFA::BaseModel
```

Comment:
Well it is a generic class, but we're not moving it to a different package yet, so I have reconsidered and the current namespace is fine. Can we add a comment like "If you like this class, please move it to a base package and rename to Sysconfig::Language" ?